src/App.tsx

```diff
@@ -4,11 +4,20 @@ import { ErrorBoundary } from "react-error-boundary";
 
```

Comment:
Please make imports look like the following. Always try to keep them aligned and don't forget to separate your own imports and imports from the library

import { Provider } from "react-redux";
import { PropsWithChildren } from "react";
import { generateTheme } from "@/theme/theme";
import { ThemeProvider } from "@emotion/react";
import { ErrorBoundary } from "react-error-boundary";

import store from "@/stores/redux/store";
import { errorFallbackRender } from "@/components/error-fallback-render/error-fallback-render";
import { EmotionCacheProvider } from "@/components/emotion-cache-provider/emotion-cache-provider";

Reply:
resolved